config/default.js

```diff
@@ -46,18 +46,6 @@ module.exports = {
     removeBeforeOpen: false,
   },
 
-  settings: {
-    defaults: {
-      mdns: {
-        // Whether or not to enable mDNS advertisements
-        enabled: true,
-
-        // Domain to advertise via mDNS
-        domain: os.hostname().split('.')[0],
-      },
-    },
-  },
```

Comment:
How should this be customised now? Can we document this somewhere?

Reply:
This is kind of the whole point. These defaults actually did nothing. They were there, and were being read, but their values never actually affected the host.

Just as before, the proper way to configure the hostname and the mDNS state is through the domain settings page in the UI.